examples/csharp/HelloPhi3V/HelloPhi3V.csproj

```diff
@@ -9,9 +9,9 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.ML.OnnxRuntimeGenAI" Version="0.3.0" Condition=" '$(Configuration)' == 'Debug' OR '$(Configuration)' == 'Release' " />
-    <PackageReference Include="Microsoft.ML.OnnxRuntimeGenAI.Cuda" Version="0.3.0" Condition=" '$(Configuration)' == 'Debug_Cuda' OR '$(Configuration)' == 'Release_Cuda' " />
-    <PackageReference Include="Microsoft.ML.OnnxRuntimeGenAI.DirectML" Version="0.3.0" Condition=" '$(Configuration)' == 'Debug_DirectML' OR '$(Configuration)' == 'Release_DirectML' " />
+    <PackageReference Include="Microsoft.ML.OnnxRuntimeGenAI" Version="0.5.0-dev" Condition=" '$(Configuration)' == 'Debug' OR '$(Configuration)' == 'Release' " />
+    <PackageReference Include="Microsoft.ML.OnnxRuntimeGenAI.Cuda" Version="0.5.0-dev" Condition=" '$(Configuration)' == 'Debug_Cuda' OR '$(Configuration)' == 'Release_Cuda' " />
```

Comment:
Should this be 0.5.1 instead?

Reply:
Yeah...this is awkward. Ideally, this should be whatever in `VERSION_INFO`. It's better to have something with `-dev` suffix, because this will differ from what's in official NuGet. We always want to test locally built packages with `-dev` suffix in the CI pipeline. But after each release, we'll update `VERSION_INFO` and sometimes it will not have `-dev` suffix like right now. I'll update this to 0.5.1 later but for the next release we still need to tweak this, in order to always test the latest version.